frontend/talentsearch/src/js/components/ClassificationDefinition/ClassificationDefinition.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { GenericJobTitleKey } from "../../api/generated";
+
+import ITDefinitions from "./ITDefinitions";
+
+interface ClassificationDefinitionProps {
+  name: string;
+}
+
+const definitionMap: Record<string, React.FC> = {
+  [GenericJobTitleKey.TechnicianIt01]: ITDefinitions.LevelOne,
+  [GenericJobTitleKey.AnalystIt02]: ITDefinitions.LevelTwo,
+  [GenericJobTitleKey.TeamLeaderIt03]: ITDefinitions.LevelThreeLead,
+  [GenericJobTitleKey.TechnicalAdvisorIt03]: ITDefinitions.LevelThreeLead,
```

Comment:
oops, just noticed this
those are connected to identical things

Reply:
Fixed with [fix it-03 advisor defintion](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3338/commits/920abbba3b07d25b7fa179d706e712f20a823381)

Reply:
Nice, think that is all from me